### PR TITLE
Add noscript tag

### LINKED ISSUE #975: Add noscript tag
tags should go in head or body.
```
<noscript> We're sorry but this page doesn't work properly without JavaScript enabled. Please enable it to continue</noscript>
```

### PR DESCRIPTION
**Pull request by rishsingh07**
fixed #975 

image

I'm unable to remove navbar.

### CODE REVIEW

File: src/features/common/Layout/Header/index.tsx [hunk 1]
```diff
@@ -96,5 +97,7 @@ export default function Header() {
         <meta name="theme-color" content={styles.primaryColor} />
       ) : null}
     </Head>
+    <noscript>This browser does not support our website</noscript>
```
Comment (norbertschuler) [inline src/features/common/Layout/Header/index.tsx:100]: Why didn't you use the wording from the issue description at https://github.com/Plant-for-the-Planet-org/planet-webapp/issues/975#issue-855395884?

**Comment by norbertschuler** [comment]
This is how the website not allowing JavaScript is looking in Chrome browser on MacOS - not nice, but at least readable:

Bildschirmfoto 2021-04-22 um 10 02 21

**Review by norbertschuler — APPROVED** [review]
There is a point missing at the end of the sentence. But otherwise this is working.

**Comment by sagararyal** [comment]
@rishsingh07 can you fix ESLint Errors.

**Comment by rishsingh07** [comment]
@sagararyal  ESLint error is fixed

**Pull request merged**